tools/loader/Loader.cpp

```diff
@@ -493,7 +493,7 @@ Loader::Loader() {
                             ExecutionBackend);
 
   hostManager_ = llvm::make_unique<runtime::HostManager>(std::move(configs));
-  backend_ = createBackend(ExecutionBackend);
+  backend_ = std::unique_ptr<Backend>(createBackend(ExecutionBackend));
```

Comment:
I'd suggest submitting this bug-fix as a small separate PR as it is a totally orthogonal issue.

Reply:
I cannot do this since the model-compiler unit test fails (ASAN).
I reverted though the ABI option modification and created a separate PR.